.github/workflows/test01-new-new.yml

```diff
@@ -0,0 +1,70 @@
+name: "test01-newParent-newChild"
+on:
+  workflow_dispatch:
+    inputs:
+      envFile:
+        description: 'Env file name'
+        required: true
+        default: 'env/.env-new-new'
+        
+env:
+  DOCKER_COMPOSE_ENV_FILE: ${{ inputs.envFile }}
+  
+jobs:
+  value-transfer-tests:
+    timeout-minutes: 360
+    runs-on: ubuntu-22.04
+    
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        
+      - name: Start containers
+        run: |
+          cd servicechain-docker
+          docker-compose -f docker-compose-servicechain.yml --env-file $DOCKER_COMPOSE_ENV_FILE up -d
+          
+      - name: Sleep 50 seconds because scn node needs to warm up during about 30 seconds
```

Comment:
When I have tested a connection with SCN by RPC or IPC call, it can not connect within 30 seconds. Therefor, sufficiently set the waiting time to 50 seconds. The sleep time is set to 30 seconds in the (docker-compose-servicechain.yml)[servicechain-docker/docker-compose-servicechain.ym] file as well. Later, after the node starts, I think it would be good to change it to a logic that checks the status and waits, and I will add this in to-do.

Reply:
Okay, please add this to the board as well.

Reply:
I added it issue #4.